### ISSUE TITLE
Update uber.com

### ISSUE THREAD

**Issue by tddybears** [jdm-contrib/jdm#2131]
## Backstory
Previously it had a yellow icon (the Just Delete Me extension) and didn't provide any info when I tried viewing it from the app. I thought I'd request an update with some info on it.

## Data

- `name`: uber.com
- `url`: https://myprivacy.uber.com/privacy/deleteyouraccount/info (I think)
- `difficulty`: Easy
- `notes`: Login -> Account / account.uber.com -> Privacy & Data / account.uber.com/data-access -> Privacy Center / privacy.uber.com/center -> Account Security -> Account Deletion -> Login again -> Continue /  myprivacy.uber.com/privacy/deleteyouraccount/info -> State a reason -> You'd be logged out
- `domains`: uber.com

**Comment by tupaschoal** [comment]
Do you still need to wait for 30 days for it to be deleted? The information currently on the website doesn't seem very different.

**Comment by tddybears** [comment]
@tupaschoal I believe so. The request was moreso to add it to the site, or what appeared to be missing at the time. When I clicked on the icon it linked to nowhere on the site (justdeleteme.xyz) and said help the project by adding it. With hindsight, it's an error of the site and not the entry, but I posted it here thinking there was technically no entry for uber.com. 

Manually searching it up appears to have uber.com show up. 
![image](https://github.com/jdm-contrib/jdm/assets/170043071/989c79c1-0dec-4bed-a9c3-f1f6359b5e7c)

But I guess the "www." makes a differences. This is the version I got when clicking on it.
![image](https://github.com/jdm-contrib/jdm/assets/170043071/2bfbba1c-b9b2-424e-8def-c61840bc189b)
